indico/modules/events/contributions/models/subcontributions.py

```diff
@@ -131,6 +131,14 @@ def timetable_entry(self):
         """Convenience property so all event entities have it"""
         return self.contribution.timetable_entry
 
+    @property
```

Comment:
Since we can't grant submission rights to speakers right now, I would leave it as it is and modify it later.